_layouts/post-toc.html

```diff
@@ -12,9 +12,16 @@ <h3>Contents</h3>
       <div class="container">
         <section class="content">{{ content }}</section>
         <footer class="prev-next">{% include prev-next.html %}</footer>
-              <hr>
-		  <p><em>Need help? <a href="https://developer.zapier.com/contact" target="_blank">Tell us about your problem</a> and we’ll connect you with the right resource or contact support.</em></p>
+          <hr>
+          <p>
+            <em>
+              Need help? 
+              <a href="https://developer.zapier.com/contact" target="_blank">Tell us about your problem</a>
+              and we’ll connect you with the right resource or contact support.
+            </em>
+          </p>
     </main>
     {% include javascript.html %}
+    {% include events-toolkit.html %}
```

Comment:
We're missing the page view event for pages using this layout. I'm adding the feature to them.